app/models/evidence_history.rb

```diff
@@ -0,0 +1,6 @@
+class EvidenceHistory < ActiveRecord::Base
+  default_scope { order('created_at DESC') }
+  belongs_to :approver, class_name: "User", foreign_key: "approved_by_user_id"
+  belongs_to :modifier, class_name: "User", foreign_key: "modified_user_id"
```

Comment:
shouldn't we call this `modified_by_user_id`, since the other one is called `approved_by_user_id`?